lib/fpm/package/deb.rb

```diff
@@ -267,6 +267,22 @@ def extract_info(package)
       #self.config_files = config_files
 
       self.dependencies += parse_depends(parse.call("Depends")) if !attributes[:no_auto_depends?]
+
+      if File.file?(File.join(path, "preinst"))
```

Comment:
You don't have to make this change, but I find lots of File.join's to be noisy, so I might consider writing this using Object#tap:

```ruby
File.join(path, "preinst").tap do |f|
  self.scripts[:before_install] = File.read(f) if File.file?(f)
end
```

We could go further and have a mapping of script symbol to file names:

{ :before_install => "preinst", ... }.each do |script, name|
  File.join(path, name).tap do |f|
      self.scripts[script] = File.read(f) if File.file?(f)
  end
end
```